newIDE/app/src/GamesShowcase/ShowcasedGameButtons.js

```diff
@@ -0,0 +1,138 @@
+// @flow
+import { Trans } from '@lingui/macro';
+import * as React from 'react';
+import GetAppIcon from '@material-ui/icons/GetApp';
+import PlayArrowIcon from '@material-ui/icons/PlayArrow';
+import YouTubeIcon from '@material-ui/icons/YouTube';
+import FlashOnIcon from '@material-ui/icons/FlashOn';
+import { sendShowcaseGameLinkOpened } from '../Utils/Analytics/EventSender';
+import Window from '../Utils/Window';
+import {
+  type ShowcasedGame,
+  type ShowcasedGameLink,
+} from '../Utils/GDevelopServices/Game';
+import { ColumnStackLayout, ResponsiveLineStackLayout } from '../UI/Layout';
+import { Spacer } from '../UI/Grid';
+import RaisedButton from '../UI/RaisedButton';
+import GooglePlay from '../UI/CustomSvgIcons/GooglePlay';
+import Steam from '../UI/CustomSvgIcons/Steam';
+import Twitter from '../UI/CustomSvgIcons/Twitter';
+import Instagram from '../UI/CustomSvgIcons/Instagram';
+import Twitch from '../UI/CustomSvgIcons/Twitch';
+import ItchIo from '../UI/CustomSvgIcons/ItchIo';
+import Microsoft from '../UI/CustomSvgIcons/Microsoft';
+import Apple from '../UI/CustomSvgIcons/Apple';
+import { useResponsiveWindowWidth } from '../UI/Reponsive/ResponsiveWindowMeasurer';
+
+const getFullUrl = url =>
+  url.startsWith('/') ? 'https://gdevelop-app.com' + url : url;
+
+const getLinkIconAndLabel = (url: string, type: string) => {
```

Comment:
should we change this to a swtich/case? I find it easier to read knowing it's based only on 1 attribute `type`